code/modules/antagonists/eldritch_cult/eldritch_transmutations.dm

```diff
@@ -177,16 +177,28 @@
 	var/mob/living/carbon/carbon_user = user
 	for(var/obj/item/living_heart/LH in atoms)
 
-		if(LH.target && LH.target.stat == DEAD)
+		if(LH.target && (LH.target.stat == DEAD || LH.target.stat == SOFT_CRIT || LH.target.stat == UNCONSCIOUS))
 			to_chat(carbon_user,span_danger("Your patrons accepts your offer.."))
 			var/mob/living/carbon/human/H = LH.target
 			H.apply_status_effect(STATUS_EFFECT_BRAZIL_PENANCE)
 			LH.target = null
 			var/datum/antagonist/heretic/EC = carbon_user.mind.has_antag_datum(/datum/antagonist/heretic)
 
-			EC.total_sacrifices++
-			EC.charge += 2
+			if(LH.target.mind.has_antag_datum(/datum/antagonist/heretic))
+				EC.total_sacrifices++
+				EC.charge += 4
+
+			if(LH.target.mind.assigned_role in GLOB.command_positions)
+				EC.total_sacrifices++
+				EC.charge += 3
+
+			if(LH.target.mind.assigned_role in GLOB.security_positions)
+				EC.total_sacrifices++
+				EC.charge += 3
 			
+			else
+				EC.total_sacrifices++
```

Comment:
also, shift EC.total_sacrifices++ out of the if/else if statements, no reason to have the line several times

Reply:
Adjusted as requested as per "I think I did this right" update